tools/update_plotlyjs.R

```diff
@@ -65,10 +65,28 @@ withr::with_dir(tmpdir, {
         file.path(lib_dir, "locales", sub("^plotly-locale-", "", basename(locales))),
         overwrite = TRUE
       )
-      # update plot schema
+      # update plot schema and (partial) bundles
+      if (!nzchar(system.file(package = "yay"))) {
+        stop("pkg 'yay' must be installed via: remotes::install_gitlab('rpkg.dev/yay')")
+      }
       Schema <- jsonlite::fromJSON(Sys.glob("dist/plot-schema.json"))
+      bundleTraceMap <-
+        yay::gh_text_file(path = "tasks/util/constants.js",
+                          owner = "plotly",
+                          name = "plotly.js",
+                          rev = basename(zip)) |>
```

Comment:
I think I'd prefer `{gh}` for this

Reply:
`yay::gh_text_file()` uses [`gh::gh_gql()`](https://gh.r-lib.org/reference/gh_gql.html) under the hood. Unfortunately, the maintainers are not up for inclusion of higher-level GraphQL functions, [I've asked](https://github.com/r-lib/gh/issues/173).

Reply:
Addendum: I've just realized that it's easy to fetch a file from a certain Git tag directly via GitHub URLs. For v2.11.1, the URL is:

https://raw.githubusercontent.com/plotly/plotly.js/v2.11.1/tasks/util/constants.js

So, no need for my yay package here. I'll update the PR.

Reply:
I've updated the PR [accordingly](https://github.com/plotly/plotly.R/pull/2336/commits/eaf3aea991b456bfbb5c820e023e56a26921e6bc) and also [added package namespace identifiers](https://github.com/plotly/plotly.R/pull/2336/commits/21c2f0e6a1659b54d84bc4b7a632fb8106edfee2) to make non-base-R dependencies more visible.

Reply:
Thank you, and sorry for the radio silence!